src/screens/Customers/CustomerMain.tsx

```diff
@@ -201,14 +205,14 @@ function CustomerMain(props: CustomerMainProps) {
         onChange={changeTab}
         variant="scrollable"
       >
-        <Tab className={classes.tab} label="All" value={CustomerStatus.ALL} />
+        <Tab className={classes.tab} label={intl(words.all)} value={CustomerStatus.ALL} />
```

Comment:
Github won't let me comment on the line, but make sure to wrap `BaseScreen`'s "Customers" in a translation